CDLUC3/dmsp_frontend_prototype
# Updated Organization Templates page to hook up with real data

## Description

- Added Graphql requests to get actual `templates` and `versionedTemplates` data from the db.
- Added error handling
- Added translations
- Added unit test
- Incorporated the new Layout container into page

Fixes # ([82](https://github.com/CDLUC3/dmsp_frontend_prototype/issues/82))

## Type of change
Update to `/template` page

## How Has This Been Tested?
 I added unit tests, did manual testing using browser, and ran `npm run build` to make sure the build wasn't breaking

## To Test
- Start this branch and the dmsp_backend_prototype branch
- Log in at `/login` page (I logged in as `super@example.com` which has an affiliationId of `https://ror.org/00dmfq477`)
- Navigate to `/template` page
- You should see a list of three templates under that user's affiliation when you come to the page
- Enter the search term `test` and click `Search` button. You should then only see the one template named 'test change'
- Click the 'x' in the Search input field to see all available templates again
- Enter a search term that will not be found, like `alfie`. You should see an error at the top of the page that that item was not found.
- Change page path to use `pt-BR` locale and note the translations

## Checklist:
- [NA] Any dependent changes have been merged and published in downstream modules
